### ISSUE TITLE
ENOENT on ember-resolver with brand new 1.13.13 app

### ISSUE THREAD

**Issue by jakemauer** [ember-cli/ember-cli#5259]
Hi there!

Before I start here are my current software versions: 
version: 1.13.13
node: 0.12.9
npm: 2.14.10
os: darwin x64

When setting up a brand new app use `ember new`, letting it install NPM and Bower dependencies, and then launching the app using `ember s` I receive the following error:

`ENOENT, no such file or directory '/Users/jake/jaketest/tmp/concat_with_maps-input_base_path-fNaE4SB5.tmp/0/bower_components/ember-resolver/dist/modules/ember-resolver.js'`

I get the same error on an another ember app on the same machine with the same ember-cli versions.

What does work, only sometimes, is deleting my `node_modeules`, `bower_components`, and `tmp` directories and running `npm cache clean && bower cache clean` and reinstalling npm and bower dependencies.

I know this is now venturing into anecdotal data, but 3 of my teammates working on our apps have encountered the same issues. Is there anything I else I can provide that will be helpful into debugging this?

Thank you for all your work.

**Comment by rwjblue** [comment]
Is ember-resolver listed in `package.json` and `bower.json`?  Just one? Which one? What versions?

**Comment by jakemauer** [comment]
In both the pre-existing and freshly created project it is listed in the bower.json as: `"ember-resolver": "~0.1.20"`

**Comment by rwjblue** [comment]
Can you confirm that `bower_components/ember-resolver/dist/modules/ember-resolver.js` is present on disk?

**Comment by rwjblue** [comment]
As you can see from https://github.com/ember-cli/ember-resolver/blob/v0.1.21/dist/modules/ember-resolver.js, it is present in the tagged repo that bower is supposed to be using...

**Comment by jakemauer** [comment]
Of course now my machine is behaving itself and `ember server` runs fine. But I believe I checked the bower components directory and the resolver was indeed there.

**Comment by jakemauer** [comment]
But I've definitely had situations recently where my bower dependencies weren't fully installing. For example I specified to use normalize-css in my bower.json, and bower install would create the normalize-css folder, but only download the Readme.md. Cloning the same project in another directory and running `bower install` installed normalize just fine. Then later doing the clean/reinstall dance, `bower install` installed normalize correctly in my original project's directory too.

**Comment by stefanpenner** [comment]
> But I've definitely had situations recently where my bower dependencies weren't fully installing.

Unfortunately this sounds like an issue with bower, we plan to move away from it entirely... the work is already on-going.

**Comment by ankushdharkar** [comment]
This https://github.com/ember-cli/ember-resolver#troubleshooting helped me.

It says : 

>As mentioned above, ember-resolver is no longer a bower package. If you're seeing a message like this:
```Unable to find a suitable version for ember-resolver, please choose one:
    1) ember-resolver#~0.1.20 which resolved to 0.1.21 and is required by ember-resolver#2.0.3
    2) ember-resolver#~2.0.3 which resolved to 2.0.3 and is required by [APP_NAME]
```
>... you probably need to update your application accordingly. See aptible/dashboard.aptible.com#423 as an example of how to update.
